templates/jvm.options.j2

```diff
@@ -67,6 +67,9 @@
 -Dio.netty.noUnsafe=true
 -Dio.netty.noKeySetOptimization=true
 -Dio.netty.recycler.maxCapacityPerThread=0
+{% if es_version | version_compare('7.4.0', '>=')  %}
```

Comment:
```suggestion
{% if es_version is version('7.4.0', '>=')  %}
```

`{% if es_version | version_compare('7.4.0', '>=')  %}` is working for our ubuntu-1804 tests which are using Ansible 2.5.1 but is failing for other tests which are using newer version of Ansible.

According to [Ansible doc](https://docs.ansible.com/ansible/latest/user_guide/playbooks_tests.html#version-comparison), the recommended syntax is now `{% if es_version is version('7.4.0', '>=')  %}`

Reply:
$ yum list ansible.noarch
	Loaded plugins: fastestmirror
	Loading mirror speeds from cached hostfile
	 * base: mirror.vcu.edu
	 * epel: iad.mirror.rackspace.com
	 * extras: mirror.vcu.edu
	 * updates: mirror.vcu.edu
	Installed Packages
	ansible.noarch		2.8.1-1.el7		@epel
	Available Packages
	ansible.noarch		2.9.1-1.el7		epel

So, I tested with 2.8.1.
Okay, thanks!